tests/e2e/e2e_workflow.spec.js

```diff
@@ -18,11 +18,10 @@ describe('Basic end-to-end Workflow', function () {
   it('Installs angular-cli correctly', function() {
     this.timeout(300000);
 
+    sh.exec('npm link', { silent: true });
```

Comment:
If I got this correctly, tests are always initiated from the root, which is why the setup with `tmp` dir works.

Although to be perfectly honest I couldn't find what the `-C` flag in `npm install` did in the command I removed.